keras/backend/tensorflow_backend.py

```diff
@@ -2086,11 +2087,8 @@ def one_hot(indices, num_classes):
         num_classes: Integer, number of classes to consider.
 
     # Returns
-        (n + 1)D one hot representation of the input
+        A tensor has (n + 1)D one hot representation of the input
```

Comment:
In that case, I'm going to pick up only `(n + 1)D one hot representation of the input...`.